src/contracts/pods/EigenPod.sol

```diff
@@ -437,7 +430,6 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         bytes calldata validatorFieldsProof,
         bytes32[] calldata validatorFields,
         bytes32[] calldata withdrawalFields,
-        uint256 beaconChainETHStrategyIndex,
         uint64 oracleTimestamp
```

Comment:
can `_verifyWithdrawalCredentials` please emit a `ValidatorBalanceUpdated` event

Reply:
> can `_verifyWithdrawalCredentials` please emit a `ValidatorBalanceUpdated` event

It doesn't seem to be as simple as I thought to add this, since the `ValidatorBalanceUpdated` event has a 'slotNumber' field, but we are not checking the slot number as part of verifying withdrawal credentials.
Unsure if there is a bug here, or what the best approach is with the event. If there's no bug in the proof-checking, then I'd ask if we can remove the 'slotNumber' field from the event.

Reply:
we need to change these to timestamps everwhere